Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealitySearchUtility.FieldSearchResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor.Search
+{
+    /// <summary>
+    /// Struct for storing search results
```

Comment:
Better comment. I know it's internal but each property also doesn't have comments.

Also maybe rename Property to Field? For some one new, SerializedProperty Property reads funny